### PR TITLE
logging messages are now printed in terminal

### PR DESCRIPTION
**Pull request by kritisingh1**
This is in reference to issue #13.
Running the command `ohpub-download` now downloads the files with logging feedback in terminal.

### CODE REVIEW

File: ohapi/public.py [hunk 1]
```diff
@@ -77,6 +77,7 @@ def download_url(result, directory, max_bytes):
                 f.write(chunk)
 
     logging.info('Downloaded {}'.format(filename))
+    return
```
Comment (madprime) [inline ohapi/public.py]: I'm unclear on why this bare return is needed or wanted here?

File: ohapi/public.py [hunk 2]
```diff
@@ -98,6 +99,7 @@ def download(source, username, directory, max_size, verbose, debug):
         logging.basicConfig(level=logging.DEBUG)
     elif verbose:
         logging.basicConfig(level=logging.INFO)
+    logging.basicConfig(level=logging.DEBUG)
```
Comment (madprime) [inline ohapi/public.py]: This is overriding the flags for `--debug` and `--verbose` that are used to set logging level. (i.e. the four lines preceding thish one.) Instead, it's forcing *all* usage of this function to produce DEBUG logging data. Why? Were you unaware of (or unable to use) the `--debug` flag to accomplish this?

Comment (kritisingh1) [reply ohapi/public.py]: `--debug` flag would print all the logging statements which are a lot of logging statements as is this PR doing currently. I understand that this should not be the appropriate behavior and wanted to discuss exactly which statements should be printed to give a minimal feedback to the user about what exactly is going on when he runs a given command. Currently not even a single message is shown in terminal which leaves the user in suspicion whether the files are being downloaded or not. and when we `ls` it through another terminal we see that multiple copies have been downloaded.

File: ohapi/public.py (hunk 1 above)
Comment (kritisingh1) [reply ohapi/public.py]: Actually, the files upon being downloaded never exited the function. So I was just trying to test the function by putting a return but it seems that multiple copies of the same files is download (I tested only for 23andme though) which is another issue. Yeah this return statement is not required here. I will remove it. :)

**Comment by madprime** [comment]
Looks good @kritisingh1, thank you!

It seems like the README.md should also get updated – I'll make a separate issue for that.

**Pull request merged**